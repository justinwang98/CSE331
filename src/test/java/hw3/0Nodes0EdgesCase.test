# Tests that a graph can be created.

# Create a graph
CreateGraph graph1

# Print the nodes in the graph
ListNodes graph1
