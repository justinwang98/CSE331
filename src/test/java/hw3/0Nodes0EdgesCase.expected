# Tests that a graph can be created.

# Create a graph
created graph graph1

# Print the nodes in the graph
graph1 contains:
